CrossCompatibility/CrossCompatibility/Utility/CompatibilityProfileLoader.cs

```diff
@@ -7,6 +7,10 @@
 using Microsoft.PowerShell.CrossCompatibility.Query;
 using CompatibilityProfileDataMut = Microsoft.PowerShell.CrossCompatibility.Data.CompatibilityProfileData;
 
+#if NETSTANDARD2_0
```

Comment:
@rjmholt The current pattern in PSSA is to define a `CORECLR` constant in the project, see e.g. [here](https://github.com/PowerShell/PSScriptAnalyzer/blob/development/Rules/Rules.csproj#L26)

Reply:
I avoided defining my own constants, since I don't know how macros cross pollute across builds (i.e. when you build from Rules.csproj vs when you build from CrossCompatibility.csproj).

I went out of my way not to build for specific PowerShell versions but instead only for .NET versions, and these constants are defined by .NET itself (they are also defined in Engine and Rules, just not used anywhere).

But since versions could change, I'll look into switching to CORECLR.

The PSV3 and PSV4 macros have been finnicky though, which is why I've avoided defining my own until now.